Synapse/Api/Room.cs

```diff
@@ -10,8 +10,19 @@ internal Room(GameObject gameObject)
             var info = gameObject.GetComponentInChildren<RoomInformation>();
             RoomType = info.CurrentRoomType;
             GameObject = gameObject;
+            LightController = null;
         }
 
+        internal FlickerableLightController LightController { get; set; }
+
+        public bool HasLightController => LightController != null;
+
+        public void Overcharge(float duration)
+            => LightController.ServerFlickerLights(duration);
```

Comment:
Either make it `LightController?.ServerSetLightIntensity(intensity);` for null checking, or leave it as it is and let it throw a NRE if `LightController` happens to be null.  
I decided to leave the `HasLightController` property as interface to tell whether or not you can do these things or not.
And this NRE to not just "swallow" it.